powerberry-app/src/services/config.py

```diff
@@ -19,10 +19,10 @@ def __init__(self):
         # query variables
         self.from_env("REDIS_HOST", default="localhost")
         self.from_env("REDIS_PORT", default=6379, cast=int)
-        self.from_cfg("influx_host", default="localhost")
-        self.from_cfg("influx_port", default=8086, cast=int)
-        self.from_cfg("voltage_reference", default=228, cast=float)
-        self.from_cfg("sampling_rate", cast=int)
+        self.from_cfg("app_influx_host", default="localhost")
+        self.from_cfg("app_influx_port", default=8086, cast=int)
+        self.from_cfg("app_voltage_reference", default=228, cast=float)
+        self.from_cfg("dsp_sampling_rate", cast=int)
```

Comment:
should we rename this into dsp_app_sampling_rate?

Reply:
I don't need it there. Changed in 4117b9f1124be2d70352360fb75fde4fadd11e66.